src/server/routes/filehandler.ts

```diff
@@ -0,0 +1,165 @@
+import express from "express";
+import { asyncHandler } from "../utils";
+import { getUserDataFromJWT } from "./utils";
+import { User, Session } from "../database/schema";
+import { v4 as uuidv4 } from "uuid";
+import { io } from "../server";
+import { Binary } from "mongodb";
+import { FileStorageType } from "../../types";
+
+export const router = express.Router();
+
+router.get(
+    "/file/:sessionId/:fileId",
+    asyncHandler<{}, { sessionId: string; fileId: string }>(
+        async (req, res) => {
+            const session = await Session.findById(req.params.sessionId);
+            const file = session?.files?.get(req.params.fileId);
+            const contents: Binary = file?.file as any;
```

Comment:
Your `Session.files` type might be wrong, it's cast to any and assigned to `Binary` here when your type is `Buffer`.